### DISCUSSION TITLE
Bug Report：can't select next station

### DISCUSSION THREAD

**Discussion by topbow** [FakeFishGames/Barotrauma#14077] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

我到了热液废土地图，发现选择下一个站点时，某些站点没有下潜的按钮，不能选择。现在我到了一个站点，相邻的有2个站点，其中一个没有下潜按钮；另一个有下潜按钮，但是选择下潜后，过一会儿又退回到地图界面，让我重新选择下一个站点。

### Reproduction steps

1.single player.
2.no use console.

[tt.zip](https://github.com/user-attachments/files/15525310/tt.zip)

![wrong](https://github.com/FakeFishGames/Barotrauma/assets/54893823/5833a35b-b121-4ab2-9657-9f065a885d61)

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Because you are already on this route

https://github.com/FakeFishGames/Barotrauma/assets/122838333/ed24883b-f3d2-4217-ac50-c2edb2f5d0b4

**Comment by topbow** [comment]
How can I go ahead to rata, without console?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "FakeFishGames/Barotrauma"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月2日(星期天) 晚上9:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [FakeFishGames/Barotrauma] Bug Report：can't select next station (Discussion #14077)





 
Because you are already on this route
 
https://github.com/FakeFishGames/Barotrauma/assets/122838333/ed24883b-f3d2-4217-ac50-c2edb2f5d0b4

**Reply by SomeRandomNoobKekeke** [reply]
just swim forward, as usual

**Comment by Regalis11** [comment]
Yeah, this doesn't seem like a bug: you cannot select the route you are already in. You can just close the map and keep going forwards.
